docs/excel/custom-functions-errors.md

```diff
@@ -1,27 +1,79 @@
 ---
-ms.date: 06/18/2019
-description: Handle errors in your Excel custom functions.
-title: Error handling for custom functions in Excel
+ms.date: 10/31/2019
+description: 'Handle and return errors like #NULL! from your custom function'
+title: Handle and return errors from your custom function (preview)
 localization_priority: Priority
 ---
 
-# Error handling within custom functions
+# Handle and return errors from your custom function (preview)
 
-When you build an add-in that defines custom functions, be sure to include error handling logic to account for runtime errors. Error handling for custom functions is the same as [error handling for the Excel JavaScript API at large](excel-add-ins-error-handling.md).
+> [!NOTE]
+> The features described in this article are currently in preview and subject to change. They are not currently supported for use in production environments. You will need to [join the Office Insider for PC](https://insider.office.com/en-us/join) to try the preview features.  A good way to try out preview features is by using an Office 365 subscription. If you don't already have an Office 365 subscription, you can get one by joining the [Office 365 Developer Program](https://developer.microsoft.com/office/dev-program).
```

Comment:
Do I have to join Office Insider _for PC_ -- or can I join other variations like Office Insider _for Mac_?  Consider removing "for PC" unless joining that specific variation of Office Insider is required (and none of the other variations are valid).

Reply:
Thanks, changed to just say Office Insider.